src/lib/crypto/kimchi_bindings/stubs/Cargo.toml

```diff
@@ -13,22 +13,22 @@ crate-type = ["lib", "staticlib"]
 
 [dependencies]
 array-init = "2.0.0"
-rmp-serde = "0.15.5"
-libc = { version = "0.2.0" }
-num-bigint = { version = "0.4.0" }
+rmp-serde = "1.1.2"
+libc = "0.2.62"
+num-bigint = { version = "0.4.4", features = [ "rand", "serde" ] }
 paste = "1.0.5"
-rand = { version = "0.8.0" }
-rayon = { version = "1" }
+rand = "0.8.5"
+rayon = "1.5.0"
 serde = "1.0.130"
-serde_json = "1.0.91"
+serde_json = "1.0.103"
 sprs = { version = "0.11.0", features = ["multi_thread"] }
 once_cell = "1.10.0"
 
 # arkworks
-ark-ff = { version = "0.3.0", features = ["parallel"] }
```

Comment:
Looks that it includes the bump up of arkworks. This should be added to the changelog for the release IMO.

Reply:
Ignore. It is 3.0.2 into master. Not 3.0.3

Reply:
Why ignore? This is 3.0.2, the current release, and it has arkworks bump. So indeed we should include it I think.

Reply:
Yeah, 3.0.2 changelog is already finalized. We were talking about 3.0.3 indeed. Closing this :)